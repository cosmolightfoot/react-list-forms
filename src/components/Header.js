import React from 'react';

export default function Header() {
  return <header>My Cool Photo Album Site</header>;
}
