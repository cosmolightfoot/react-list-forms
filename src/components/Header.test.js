import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('header test', () => {
  it('header snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
