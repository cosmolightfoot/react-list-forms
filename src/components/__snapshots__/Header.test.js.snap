// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test header snapshot 1`] = `
<header>
  My Cool Photo Album Site
</header>
`;
